accounts-db/src/accounts_db.rs

```diff
@@ -7604,9 +7607,13 @@ impl AccountsDb {
             |accum: &DashMap<Pubkey, (AccountHash, AccountSharedData)>,
              loaded_account: LoadedAccount| {
                 // Storage may have duplicates so only keep the latest version for each key
+                let mut loaded_hash = loaded_account.loaded_hash();
```

Comment:
I don't think this is necessary.
https://github.com/anza-xyz/agave/blob/bc7b27472f2457bb5a13a3e980b7b5ca498f4e31/accounts-db/src/accounts_db.rs#L898

https://github.com/anza-xyz/agave/blob/bc7b27472f2457bb5a13a3e980b7b5ca498f4e31/accounts-db/src/accounts_cache.rs#L140-L151

Reply:
Can you elaborate more on this? Is this lambda fn is called on a storage, i.e. https://github.com/anza-xyz/agave/blob/bc7b27472f2457bb5a13a3e980b7b5ca498f4e31/accounts-db/src/accounts_db.rs#L4878?